x/capability/keeper/keeper.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/capability/types"
 )
 
+var initialized = false
+
```

Comment:
Let's add a very explicit godoc for `initialized` -- why it's needed and how it's used and how it's a temporary, not ideal solution.